tsconfig.json

```diff
@@ -12,6 +12,7 @@
       "target": "es5"
     },
     "include": [
-      "src/**/*"
+      "src/**/*",
+      "test/**/*"
```

Comment:
nit: we already ahve another tsconfig that does this: `test/tsconfig.json` (but the `outDir` is a bit different); might make sense to consolidate the two sometime in the future